qr-code-auth/README.md

```diff
@@ -1,59 +1,310 @@
+  Status: 200 OK<br/>
+  ```json
+  {
+    "message": "Authentication document retrieved successfully.",
+    "count": 2,
+    "data": [
+      {
+        "user_id": "SooJK37gzjIZfFNH0tlL",
+        "device_info": "t5k77PHnuDSrgEzvMJAj",
+        "is_authorized": true,
+        "access_token": "NLFSj7Kz30oHgolfIZtJ"
+      },
+      {
+        "user_id": "SooJK37gzjIZfFNH0tlL",
+        "device_info": "en7nlNnpfqoqodcmtMeZ",
+        "is_authorized": false,
+        "access_token": "GTB4UUtlKwGemRN2lwBp11"
+      }
+    ]
+  }
+  ```
+  GET /qr-code-auth?user_id=invalidUserId<br/>
+  Status: 404 Not Found<br/>
+  ```json
+  {
+    "message": "No Authentication authentication found."
+  }
+  ```
+  GET /qr-code-auth?taskId=GTB4UUtlKwGemRN2lwBp11<br/>
+  Status: 400 Bad Request
+  ```json
+  {
     "statusCode": 400,
-    "message": "Bad Request"
-}
-```
\ No newline at end of file
+    "error": "Bad Request",
+    "message": "invalid query parameters passed"
+  }
+  ```
+  GET /?user_id=GTB4UUtlKwGemRN2lwBp11
+  Status: 500 Internal Server Error
+  ```json
+  {
+    "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+  }
+  ```
+
+Sure, here's the updated API contract for the PATCH call:
+
+## PATCH /qr-code-auth/user_id/{user_id}/is_authorized/{is_authorized}
+
+Updates the is_authorized field of an existing qr-code-auth document for the specified user.
+
+- **Params**
+  - **user_id** (required, string): Specifies the user ID associated with the authentication document.
+  - **is_authorized** (required, boolean): Specifies whether the authentication document is authorized or not.
+- **Query**
+  None
+- **Headers**
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "message": "String",
+        "data": {
+          "user_id": "String",
+          "device_info": "String",
+          "is_authorized": "Boolean",
+          "access_token": "String"
+        }
+      }
+      ```
+- **Error Response:**
+
+  - **Code:** 400
+
+    - **Content:**
+      ```json
+      {
+        "message": "Bad Request"
+      }
+      ```
+
+  - **Code:** 401
+    - **Content:**
+      ```json
+      {
+        "statusCode": 401,
+        "error": "Unauthorized",
+        "message": "Unauthenticated User."
+      }
+      ```
+  - **Code:** 404
+    - **Content:**
+      ```json
+      {
+        "message": "Document not found"
+      }
+      ```
+  - **Code:** 500
+    - **Content:**
+      ```json
+      {
+        "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+      }
+      ```
+
+- **Example for updating an existing authentication document:**
+  PATCH /qr-code-auth/user_id/SooJK37gzjIZfFNH0tlL/is_authorized/true<br/>
+  Content-Type: application/json<br/>
+  Request-Body:<br/>
```

Comment:
Who can call this API?

Reply:
> Why are events, monitor, and progress changes here?

Since the target branch is not updated with the latest main we see those file changes. I have asked @shreya-mishra  to take a latest pull from main to resolve this.

Reply:
I discussed with @shreya-mishra. As per the discussion the users can call this API.